site/src/components/WorkspaceOutdatedTooltip/WorkspaceOutdatedTooltip.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
+import { useQueryClient } from "react-query";
+import { MockTemplateVersion, MockTemplate } from "testHelpers/entities";
+import { WorkspaceOutdatedTooltip } from "./WorkspaceOutdatedTooltip";
+
+const meta: Meta<typeof WorkspaceOutdatedTooltip> = {
+  title: "components/WorkspaceOutdatedTooltip",
+  component: WorkspaceOutdatedTooltip,
+  decorators: [
+    (Story) => {
+      const queryClient = useQueryClient();
+      queryClient.setQueryData(
+        ["templateVersion", MockTemplateVersion.id],
+        MockTemplateVersion,
+      );
+      return <Story />;
+    },
+  ],
+  args: {
+    onUpdateVersion: action("onUpdateVersion"),
+    templateName: MockTemplate.display_name,
+    latestVersionId: MockTemplateVersion.id,
+  },
```

Comment:
Instead of adding a new decorator you can use parameters like:
```tsx
parameters: {
  queries: [
    {
      key: ["templateVersion", MockTemplateVersion.id],
      data: MockTemplateVersion 
    }
  ]
}
```

Reply:
oh! neat! the health page example I looked at used `useQueryClient`, but this seems nice